src/db/z_db.erl

```diff
@@ -847,6 +860,7 @@ merge_props([R|Rest], Acc) ->
     end.
 
 
+-spec assoc1(atom(), #context{}, string(), [tuple()], pos_integer()) -> {ok, [[tuple()]]}.
 assoc1(DbDriver, C, Sql, Parameters, Timeout) ->
     case DbDriver:equery(C, Sql, Parameters, Timeout) of
```

Comment:
Remove the dynamic function call on line 865. Only modules that define callbacks should make dynamic calls.